Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -373,7 +374,7 @@ def generate_search_query(time_to_fetch_from: Optional[datetime],
     Returns:
         A list with arguments for the email search query
     """
-    LOG('in generate search query')
+    demisto.debug('in generate search query')
```

Comment:
use the [logger function](https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L1679) instead